README.md

```diff
@@ -1,37 +1,67 @@
 # flutter_template_riverpod
 
-MonstarLab Flutter Template w/Riverpod 2.x
+MonstarLab Flutter Template w/Riverpod
 
-## Premise
+## Getting Started
 
-Based loosely around the
-prior [MonstarLab Flutter Template](https://github.com/monstar-lab-oss/flutter-template) which uses
-BLOC instead of Riverpod. The main theme is Anime News. The API used
-is [Jikan API (4.0.0)](https://docs.api.jikan.moe) which is an unofficial API for the well
-known [MyAnimeList](https://myanimelist.net/) website. No API key is required as of creation, making
-it ideal for a demo/template.
+This project is a starting point for a Flutter application.
 
-## Concepts demonstrated
+A few resources to get you started if this is your first Flutter project:
 
-- [Flutter Riverpod Architecture](https://codewithandrea.com/articles/flutter-app-architecture-riverpod-introduction/)
-  - Presentation, Application, Domain and Data Layers.
-- Freezed Code Generation
-- Riverpod Provider Generation
-- Retrofit API calls, `toJson` function Generation.
+- [Lab: Write your first Flutter app](https://docs.flutter.dev/get-started/codelab)
+- [Cookbook: Useful Flutter samples](https://docs.flutter.dev/cookbook)
 
-## How to run
+For help getting started with Flutter development, view the
+[online documentation](https://docs.flutter.dev/), which offers tutorials, samples, guidance on
+mobile development, and a full API reference.
 
+## How to build
 - run below command
-
 ```
 flutter pub get
 flutter packages pub run build_runner build --delete-conflicting-outputs
 ```
 
-### Visual Studio
-
-Launch Configurations are included in the `.vscode/launch.json` file.
+## Run 
+### Run iOS/Android using Visual Studio
+- create launch.json and add add below
+```
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "flutter_template_riverpod(development mode)",
+      "request": "launch",
+      "type": "dart",
+      "program": "lib/main/main_development.dart",
+      "args": [
+        "--flavor",
+        "Development",
+       ]
+    },
+    {
+      "name": "flutter_template_riverpod (release mode)",
+      "request": "launch",
+      "type": "dart",
+      "flutterMode": "release",
+      "program": "lib/main/main_production.dart",
+      "args": [
+        "--flavor",
+        "Release",
+      ]
+    }
+  ]
+}
+```
 
-### Android Studio
+### run iOS/Android From terminal
+```
+- flutter run --flavor Development -t lib/main/main_development.dart
+```
 
-Launch Configurations are included in the `.run` folder. 
\ No newline at end of file
+### run iOS using xcode
+- Select Development schema from top center and run in development mode(Debug)
+- Should select Production for archive release mode ipa file
```

Comment:
The README changes are completely unrelated to the environments.  Can we better handle this in another PR?

Reply:
i think we can do in same pr because i wrote about what changes i did.
also if you think any suggestion i can update read me.

Reply:
You threw away all of the comments in my old Readme file which explained the basis for creating this template, and replaced it with the default Readme template?! You also removed the Android Studio section.
I will not approve these changes to the Readme unless you restore those parts.

Reply:
hey sorry it was by mistakes seems some temporary mistakes i fixed.